main.go

```diff
@@ -85,6 +85,10 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "OSBuildConfig")
 		os.Exit(1)
 	}
+	if err = (&osbuilderprojectflottaiov1alpha1.OSBuildConfig{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
in flotta for simplify dev-env, webhooks are disabled by default when running `make run` by exposing env-var.
see https://github.com/project-flotta/flotta-operator/blob/main/main.go#L132